src/Components/HeaderControlButtonWithCallout/HeaderControlButtonWithCallout.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+    IHeaderControlButtonWithCalloutProps,
+    IHeaderControlButtonWithCalloutStyleProps,
+    IHeaderControlButtonWithCalloutStyles
+} from './HeaderControlButtonWithCallout.types';
+import { getStyles } from './HeaderControlButtonWithCallout.styles';
+import {
+    classNamesFunction,
+    useTheme,
+    styled,
+    FocusTrapCallout,
+    FontIcon,
+    IconButton,
+    Stack
+} from '@fluentui/react';
+import { useBoolean, useId } from '@fluentui/react-hooks';
+import HeaderControlButton from '../HeaderControlButton/HeaderControlButton';
+import HeaderControlGroup from '../HeaderControlGroup/HeaderControlGroup';
+
+const getClassNames = classNamesFunction<
+    IHeaderControlButtonWithCalloutStyleProps,
+    IHeaderControlButtonWithCalloutStyles
+>();
+
+const HeaderControlButtonWithCallout: React.FC<IHeaderControlButtonWithCalloutProps> = (
+    props
+) => {
+    const { buttonProps, calloutProps, children, styles } = props;
+
+    // hooks
+    const calloutAnchor = useId();
+
+    // state
+    const [showPicker, { toggle: togglePicker }] = useBoolean(false);
```

Comment:
lets change the name reflecting it is a bool like isPickerVisible

Reply:
Done.